### ISSUE TITLE
[Bug] User-Defined Calibration Not Affecting Measurements in Cornerstone Viewport

### ISSUE THREAD

**Issue by tokyobordel** [cornerstonejs/cornerstone3D#1711]
### Describe the Bug

Calibration does not change the values ​​of drawn annotations. My code:
```
const calibration = {
  scale: 2.0,
  type: cornerstoneCore.Enums.CalibrationTypes.USER
};

cornerstoneTools.utilities.calibrateImageSpacing(
  viewport.getCurrentImageId(),
  viewport.getRenderingEngine(),
  calibration
) 
```
In theory, after executing this code, a value should be written to the "calibration" field of stack viewport instance, but I still have undefined. 
Version on Cornerstone3D: 2.12.0

### Steps to Reproduce

```
const calibration = {
  scale: 2.0,
  type: cornerstoneCore.Enums.CalibrationTypes.USER
};

cornerstoneTools.utilities.calibrateImageSpacing(
  viewport.getCurrentImageId(),
  viewport.getRenderingEngine(),
  calibration
) 
```

### The current behavior

Calibration does not change the values ​​of drawn annotations.

### The expected behavior

The values ​​of the drawn annotations have changed according to the calibration parameters.

### OS

Windows 10

### Node version

20.15.1

### Browser

Chrome 130.0.6723.70

**Comment by sedghi** [comment]
Does this example work? https://www.cornerstonejs.org/live-examples/calibrationtools

maybe see the code to see how we do it

**Comment by sedghi** [comment]
Any update?

**Comment by tokyobordel** [comment]
Yes it works now. It was my mistake, I forget about calibratedPixelSpacingMetadataProvider
